### PR TITLE
Data Privacy content updates

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
- Update Privacy Notices sidebar block header and copy
- Update Get Involved sidebar block copy
- Add 'Transparency' sidebar block
- Archive the following privacy notices:
  - Firefox + Cliqz
  - Firefox Marketplace
- Delete Cloud Services link (just goes to Firefox policy)
- Add Firefox Fire TV link

## Localization

- The index pages is localized and has new strings behind a tag.
- The policies are archived in English only. 

## Issue / Bugzilla link
Closes mozilla/bedrock#5745

### CODE REVIEW

**Comment by stephaniehobson** [comment]
String change coming for Get Involved sidebar.

**Comment by stephaniehobson** [comment]
String change coming tomorrow.

**Comment by stephaniehobson** [comment]
Oop, should have said, string change is done.

File: bedrock/privacy/templates/privacy/archive/firefox-cliqz-2018-06.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "privacy/base-resp.html" %}
+
+{% block page_title %}Mozilla Firefox + Cliqz Privacy Notice — Archived, June 2018{% endblock %}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/privacy/templates/privacy/archive/firefox-cliqz-2018-06.html]: @ejregithub This is a good question, who do we need to ask?

File: bedrock/privacy/templates/privacy/index.html [hunk 2]
```diff
@@ -69,22 +69,26 @@ <h2>{{ _('Data Privacy Principles') }}</h2>
...
+          We created short and clear Privacy Notices to describe how each of our products and services
+          receives, shares, and uses data and what your choices are. Learn more:
+        {%- endtrans -%}
+      {% endif %}
     </p>
     <nav>
       <ul>
         <li class="policy-websites"><a href="{{ url('privacy.notices.websites') }}">{{ _('Mozilla Websites, Communications &amp; Cookies') }}</a></li>
         <li class="policy-firefox"><a href="{{ url('privacy.notices.firefox') }}">{{ _('Firefox Browser') }}</a></li>
+        <li class="policy-firefox"><a href="https://www.mozilla.org/privacy/firefox-fire-tv/">{{ _('Firefox for Fire TV') }}</a></li>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/privacy/templates/privacy/index.html]: I should write a script to make sure I'm not putting fully qualified URLs into things 😣 

I've been told there will eventually be a {{ url('privacy.notices.firefox-fire-tv') }} page and I worry that linking here to focus/klar directly will mean this link doesn't get updated when that is created. Is it possible to create a `url()` link to a redirect?

**Comment by stephaniehobson** [comment]
DNM until July 3 now.

File: bedrock/privacy/templates/privacy/index.html [hunk 3]
```diff
@@ -69,22 +69,26 @@ <h2>{{ _('Data Privacy Principles') }}</h2>
     </p>
   </section>
   <section id="side-notices">
-    <h2>{{ _('Our Privacy Notices') }}</h2>
+    <h2>{{ _('Privacy Notices') }}</h2>
```
Comment (flodolo) [inline bedrock/privacy/templates/privacy/index.html]: You can't change an existing string without a tag.

File: bedrock/privacy/templates/privacy/index.html [hunk 4]
```diff
@@ -97,20 +101,51 @@ <h2>{{ _('Our Privacy Notices') }}</h2>
       </ul>
     </nav>
   </section>
+  {% if l10n_has_tag('updates-052018') %}
+    <section id="side-transparency">
+      <h2>{{ _('Transparency Report ') }}</h2>
```
Comment (flodolo) [inline bedrock/privacy/templates/privacy/index.html]: Extra trailing space

File: bedrock/privacy/templates/privacy/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/privacy/templates/privacy/index.html]: Once 5833 merges, this conundrum goes away :)

**Comment by stephaniehobson** [comment]
- added a test for the redirect
- remove the trailing space from one string
- added tag around another changed string

@flodolo I made the requested changes and amended them to the commit before remembering that you prefer to have string changes made as subsequent commits. Sorry. The strings are ready for extraction - however - Alex will have to take over this PR and add a German version of the archived Cliqz privacy policy. It shouldn't change these strings but I don't know what else it might add or how that will be achieved.

**Comment by stephaniehobson** [comment]
Oh and I rebased it after merging #5833.

**Comment by alexgibson** [comment]
Closing in favor of https://github.com/mozilla/bedrock/pull/5850

**Pull request closed without merging**